stacks-common/src/util/vrf.rs

```diff
@@ -297,8 +299,8 @@ impl VRFProof {
 
                 c_buf[..16].copy_from_slice(&bytes[32..(16 + 32)]);
                 s_buf[..32].copy_from_slice(&bytes[48..(32 + 48)]);
-                let c = ed25519_Scalar::from_canonical_bytes(c_buf)?;
-                let s = ed25519_Scalar::from_canonical_bytes(s_buf)?;
+                let c = ed25519_Scalar::from_canonical_bytes(c_buf).expect("Invalid C scalar");
+                let s = ed25519_Scalar::from_canonical_bytes(s_buf).expect("Invalid S scalar");
```

Comment:
```suggestion
                let c = ed25519_Scalar::from_canonical_bytes(c_buf).ok()?;
                let s = ed25519_Scalar::from_canonical_bytes(s_buf).ok()?;
```

Reply:
@kantai 
Actually, this doesn't work. `CtOption` is kind of a weird type, doesn't implement  `.ok()` 

this could work
```rust
                let c = ed25519_Scalar::from_canonical_bytes(c_buf).map(|_| {
                    test_debug!("Invalid C scalar");
                    return None;
                });
                let s = ed25519_Scalar::from_canonical_bytes(s_buf).map(|_| {
                    test_debug!("Invalid S scalar");
                    return None;
                });
```

What do you think?

Reply:
I don't really understand the point of `CtOption` type.
It's like an Option but different.

I made this instead:
```rust
                let c: Option<ed25519_Scalar> = ed25519_Scalar::from_canonical_bytes(c_buf).into();
                let s: Option<ed25519_Scalar> = ed25519_Scalar::from_canonical_bytes(s_buf).into();

                Some(VRFProof {
                    Gamma: gamma,
                    c: c?,
                    s: s?,
                })
```

(It's a Result so `.ok()` isn't available)